genomics/4-Variation/fetch_sra_acc.sh

```diff
@@ -17,6 +17,12 @@ curl -s 'https://www.ebi.ac.uk/ena/browser/api/xml/links/taxon?accession=2697049
 
 cat ena.txt genbank.txt sra.txt | sort | uniq > union.txt
 rm ena.txt
+
+if [ ! -f acc2exclude.txt ]
```

Comment:
I guess you can just do touch without the if as touch only updates the access times.

Reply:
You are right. Didn't think about it. Shall I do a new PR?

Reply:
I guess you can do it here directly ...